specs/eventing/channel.md

```diff
@@ -152,37 +156,20 @@ Each instantiated Channel (ie, Custom Object) SHOULD have an annotation
 indicating which version of the `Channelable` duck type it conforms to. We
 currently have these versions:
 
-1. [v1beta1](https://github.com/knative/eventing/blob/main/pkg/apis/duck/v1beta1/channelable_types.go)
 1. [v1](https://github.com/knative/eventing/blob/main/pkg/apis/duck/v1/channelable_types.go)
 
-So, for example to indicate that the Channel supports v1beta1 duck type, you
-should annotate it like so (only showing the annotations):
+So, for example to indicate that the Channel supports v1 channelable duck type,
+you should annotate it like so (only showing the annotations):
 
 ```
-- apiVersion: messaging.knative.dev/v1beta1
+- apiVersion: messaging.knative.dev/v1
   kind: YourChannelType
   metadata:
-    annotations: messaging.knative.dev/subscribable: v1beta1
+    annotations: messaging.knative.dev/subscribable: v1
 ```
 
-Unfortunately, we had to make breaking changes between v1alpha1 and v1beta1
-versions, and to ensure functionality, the channel implementer must indicate
-which version they support. To ensure backwards compatibility with old channels,
-if no annotation is given, we assume it's `v1alpha1`.
-
 #### Spec Requirements
 
-##### v1beta1 Spec
-
-Each channel CRD MUST contain an array of subscribers:
-[`spec.subscribers`](https://github.com/knative/eventing/blob/main/pkg/apis/duck/v1beta1/subscribable_types.go)
-
-Note: The array of subscribers MUST NOT be set directly on the generic Channel
-custom object, but rather appended to the backing channel by the subscription
-itself.
-
-##### v1 Spec
-
```

Comment:
Dropping this is what I mean by there's no versioning. I think I'd rather leave the v1 in right now, at least to indicate that there is a versioning story and things could change in the future, but the v1 is the spec we support. But see the comment about having v1beta1 noted and supported for impls that are out there and might not advertise themselves as supporting v1 but in fact are totes wire compatible with the v1 and we should continue to support them. Maybe it's something that's not necessary for the spec to the letter of the law, but I think it might be nice to state for the existing things.